src/plugins/management/public/types.ts

```diff
@@ -78,3 +78,9 @@ export interface CreateManagementItemArgs {
   capabilitiesId?: string; // overrides app id
   redirectFrom?: string; // redirects from an old app id to the current app id
 }
+
+export interface ClientConfigType {
```

Comment:
**question** why does the `management` plugin need this information? My knowledge is admittedly outdated, but I thought this plugin tried to be ignorant to the individual sub-applications contained within.

Reply:
I agree, it would indeed be better if we abstract `management` from the config type for the security apps. Moved the declaration of the type to `x-pack/plugins/security/public/management/management_service.ts`. Let me know if you have a better suggestion.

Reply:
That works for me, thanks